client/app/scripts/components/plugins.js

```diff
@@ -1,10 +1,20 @@
 import React from 'react';
+import classNames from 'classnames';
 
 export default class Plugins extends React.Component {
-  renderPlugin(plugin) {
+  renderPlugin({id, label, description, status}) {
+    const error = status !== 'ok';
+    const className = classNames({ error });
+    const title = error ?
+      `Status: ${status}. (Plugin description: ${description})` :
```

Comment:
Yes, agree! Good suggestion!

On 13 April 2016 at 11:45, David <notifications@github.com> wrote:

> In client/app/scripts/components/plugins.js
> <https://github.com/weaveworks/scope/pull/1280#discussion_r59520620>:
>
> >
> >  export default class Plugins extends React.Component {
> > -  renderPlugin(plugin) {
> > +  renderPlugin({id, label, description, status}) {
> > +    const error = status !== 'ok';
> > +    const className = classNames({ error });
> > +    const title = error ?
> > +      `Status: ${status}. (Plugin description: ${description})` :
>
> +1, i think it's good seeing that things are ok.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/weaveworks/scope/pull/1280/files/7e5302afbb16872a971abbf025754106d550484b#r59520620>
>